client/src/component/CodeWarsCard.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+export const CodeWarsCard = ({ rank, progress, milestone }) => {
```

Comment:
Check this line, because when i ran your code it was an error, because of this export const, i don't know why. You can just copy and paste: export const CodeWarsCard = ({ rank, progress, milestone }) => {